<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scheduler Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="type_of_scheduler_label">
        <property name="text">
         <string>Type of Scheduler:</string>
        </property>
        <property name="buddy">
         <cstring>type_of_scheduler_comboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type_of_scheduler_comboBox">
        <item>
         <property name="text">
          <string>FCFS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SJF (Preemptive)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SJF (Non Preemptive)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Priority (Preemptive)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Priority (Non Preemptive)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Round Robin</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="no_of_processes_label">
        <property name="text">
         <string>Number of Processes:</string>
        </property>
        <property name="buddy">
         <cstring>no_of_processes_spinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="no_of_processes_spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="quantum_label">
        <property name="text">
         <string>Quantum:</string>
        </property>
        <property name="buddy">
         <cstring>quantum_doubleSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="quantum_doubleSpinBox">
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="enter_data_pushButton">
      <property name="text">
       <string>Enter Process Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
